build/cmake/functions.cmake

```diff
@@ -706,6 +706,17 @@ function(wx_add_thirdparty_library var_name lib_name help_str)
 endfunction()
 
 function(wx_print_thirdparty_library_summary)
+  # Avoid printing out the message if we're being reconfigured and nothing has
+  # changed since the previous run, so check if the current summary differs
+  # from the cached value.
+  foreach(entry IN LISTS wxTHIRD_PARTY_LIBRARIES)
+    # TODO-CMAKE-VERSION: Use list(JOIN) with CMake 3.12+.
+    string(APPEND wxTHIRD_PARTY_SUMMARY_NOW "${entry}-")
+  endforeach()
+
+  if(NOT "${wxTHIRD_PARTY_SUMMARY_NOW}" STREQUAL "$CACHE{wxTHIRD_PARTY_SUMMARY}")
```

Comment:
`$CACHE{VAR}` is added in version 3.13. I don't think it is needed here and you can just use `"${wxTHIRD_PARTY_SUMMARY}"`

Reply:
Thanks, I didn't notice this and I don't know why did I use `$CACHE{}` originally... I think I wanted to make sure I was using the original value of the variable, but it looks like it's not really necessary, so I've just removed it, thanks.